vendor/bat-native-ads/README.md

```diff
@@ -30,6 +30,12 @@ Initialize Ads by calling `Initialize` when Ads are enabled or disabled on the C
 void Initialize()
 ```
 
+`IsSupportedRegion` should be called to determine if Ads are supported for the specified region
+```
+stastic bool IsSupportedRegion(
```

Comment:
Typo: stastic

Reply:
👍 done. Did you not know this was coming in c++20